assets/inbuilt/transformers/dockerfile/winconsole/winconsole.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: move2kube.konveyor.io/v1alpha1
+kind: Transformer
+metadata:
+  name: WinConsoleApp-Dockerfile
+spec:
+  mode: "Container"
+  class: "WinConsoleAppDockerfileGenerator"
+  generates: 
```

Comment:
The generates and generatesbases fields are no longer part of the yaml. We can remove them